### ISSUE TITLE
Layertree with thematic layer true - Layersets are not displayed in 3.2.4

### ISSUE THREAD

**Issue by astroidex** [mapbender/mapbender#1330]
**Describe the bug**
When you create a new layertree element and activate thematic layer it is not displayed. In older application that were saved with version < 3.2.x everything is still fine.

**ERROR**
Newer Mapbender versions save the layerset-ID as integer and not as string anymore (see screenshot). In the frontend the layerset titles are not displayed anymore, when the layerset-ID is saved as integer. 

![Bildschirmfoto vom 2021-03-15 17-40-53](https://user-images.githubusercontent.com/996298/111193937-012c6f80-85bb-11eb-8b87-6cb7ba8acaf3.png)

This is how it should look like:
![layertree_thematic_layertree_frontend](https://user-images.githubusercontent.com/996298/111194125-291bd300-85bb-11eb-9b9f-bfce16a2b792.png)

![layertree_thematic_layertree](https://user-images.githubusercontent.com/996298/111193913-fa9df800-85ba-11eb-83b1-87124986749d.png)

**To Reproduce**
1. Import mapbender_user_basic to your database
2. go to the layertree element and activate thematic layer -> reopen the element again and save
3. See error in the application: no layerset titles are displayed

**Expected behavior**
Should work for all Mapbender version

**Versions (please complete the following information if it helps to narrow down the bug):**
 - Mapbender: 3.2.4

```
new definition - does NOT display the thematic layertree
-> saves layerset id as integer
a:10:{s:6:"target";s:4:"1497";s:4:"type";s:7:"element";s:8:"autoOpen";b:0;s:14:"showBaseSource";b:1;s:10:"hideSelect";b:0;s:8:"hideInfo";b:0;s:4:"menu";a:0:{}s:8:"useTheme";b:1;s:12:"allowReorder";b:1;s:6:"themes";a:2:{i:0;a:4:{s:2:"id";i:149;s:6:"opened";b:1;s:5:"title";s:4:"main";s:8:"useTheme";b:1;}i:1;a:4:{s:2:"id";i:151;s:6:"opened";b:1;s:5:"title";s:9:"layerset2";s:8:"useTheme";b:1;}}}


old definition - still works fine - displays the thematic layertree
-> saved layerset id as charcter
a:10:{s:6:"target";s:4:"1497";s:4:"type";s:7:"element";s:8:"autoOpen";b:0;s:14:"showBaseSource";b:1;s:10:"hideSelect";b:0;s:8:"hideInfo";b:0;s:4:"menu";a:0:{}s:8:"useTheme";b:1;s:12:"allowReorder";b:1;s:6:"themes";a:2:{i:0;a:4:{s:2:"id";s:3:"149";s:6:"opened";b:1;s:5:"title";s:4:"main";s:8:"useTheme";b:1;}i:1;a:4:{s:2:"id";s:3:"151";s:6:"opened";b:1;s:5:"title";s:9:"layerset2";s:8:"useTheme";b:1;}}}
```

**Comment by thzanger** [comment]
Dear @astroidex, thank you for the information. I forwarded the information to out internal MB ticket system.

**Comment by thzanger** [comment]
Dear @astroidex, the problem has now been fixed for the latest version of 3.2.4. Regards
